server/server.py

```diff
@@ -37,7 +37,8 @@
         attn_implementation="flash_attention_2" if is_flash_attn_2_available() else "sdpa",
         trust_remote_code=True, 
         low_cpu_mem_usage=True, 
-        torch_dtype=dtype),
+        torch_dtype=dtype,
+        device_map="auto"),
```

Comment:
this splits the model across all available devices. this means that multiple lower vram devices can be used to be able to inference the model